### PR TITLE
update help.txt

### PR DESCRIPTION
**Pull request by SolaNoVa**
Update help.txt with commands in the messages for the boat to dm people with the http://docs.frederikam.com still can be found on top of the post, tested in discord it will fit :)

Regards
Alyzza :D

### CODE REVIEW

File: FredBoat/src/main/resources/help.txt [hunk 1]
```diff
@@ -1,6 +1,35 @@
 Documentation can be found at the bot's GitHub pages:
 http://docs.frederikam.com
 
+Commands
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: You have to make it clear that the list of commands is non-exhaustive.

File: FredBoat/src/main/resources/help.txt [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
+```
+Main
+    ;;help = Sends a PM with instructions
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: With Discord, the correct term is usually `DM`

File: FredBoat/src/main/resources/help.txt [hunk 3]
```diff
@@ -1,6 +1,35 @@
...
+    ;;say <text> =  Make the bot echo something
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: *makes

File: FredBoat/src/main/resources/help.txt [hunk 4]
```diff
@@ -1,6 +1,35 @@
...
+    ;;avatar = Displays the avatar of a user
+    ;;brainfuck <code> [input] = Executes Brainfuck code
+    ;;lua <code> = Executes Lua code
+    ;;riot <text> = Transform text, example
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: *transforms text
Which example?

File: FredBoat/src/main/resources/help.txt [hunk 5]
```diff
@@ -1,6 +1,35 @@
...
+    ;;lenny = Sends a lenny face
+    ;;leet <text> = Makes you sound like a script kiddie
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: ;;leet command is broken and should be removed

File: FredBoat/src/main/resources/help.txt [hunk 6]
```diff
@@ -1,6 +1,35 @@
...
+    ;;mal <search term> = Searched MyAnimeList for animes and users
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: *Searches

File: FredBoat/src/main/resources/help.txt [hunk 7]
```diff
@@ -1,6 +1,35 @@
...
+    ;;dump <1-2000> = Dumps between 1 and 2000 messages to Hastebin
+    ;;facedesk = Uploads an image
+    ;;roll = Uploads an image ```
+```
+Music
+    ;;play <url> = Plays music from the given URLs. See supported sources below
+    ;;list = Displays a list of the current songs in the playlist
+    ;;nowplaying = Displays the currently playing song
+    ;;skip = Skip the current song. Please use in moderation
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: *Skips

File: FredBoat/src/main/resources/help.txt [hunk 8]
```diff
@@ -1,6 +1,35 @@
...
+    ;;stop = Stop the player and clear the playlist. Reserved for moderators.
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: *Stops

File: FredBoat/src/main/resources/help.txt [hunk 9]
```diff
@@ -1,6 +1,35 @@
...
+    ;;pause = Pause the player.
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: *Pauses

File: FredBoat/src/main/resources/help.txt [hunk 10]
```diff
@@ -1,6 +1,35 @@
...
+    ;;unpause = Unpause the player.
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: *Unpauses

File: FredBoat/src/main/resources/help.txt [hunk 11]
```diff
@@ -1,6 +1,35 @@
...
+    ;;repeat = Toggles repeat mode for the current song.
+    ;;leave = Makes the bot leave the current voice channel
+    ;;shuffle = Toggles shuffle mode for the current queue.
+    ;;volume = Changes the volume. Values are 0-150 and 100 is the default.```
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: ;;volume is deprecated

File: FredBoat/src/main/resources/help.txt [hunk 12]
```diff
@@ -1,6 +1,35 @@
...
+
```
Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: No ;;fwd, ;;rew and ;;seek commands

File: FredBoat/src/main/resources/help.txt (hunk 1 above)
Comment (SolaNoVa) [inline FredBoat/src/main/resources/help.txt]: what do you mean by non-exhaustive?

Comment (Frederikam) [inline FredBoat/src/main/resources/help.txt]: That this list is incomplete (and that's intentional)

**Comment by Frederikam** [comment]
I am closing this PR because the original repo has been deleted, therefore changes to this PR is impossible. Please re-create this PR from your new repo.

**Pull request closed without merging**